src/engraving/libmscore/layoutbreak.h

```diff
@@ -72,6 +72,10 @@ class LayoutBreak final : public EngravingItem
 
     void layout0();
 
+    double lw() const { return m_lw; }
```

Comment:
It might be even nicer to call this `lineWidth`

Reply:
Thought about tidying up when I will add LayoutData. But it's better to do simple things right away, I agree

Reply:
renamed